client/views/chat.html

```diff
@@ -0,0 +1,44 @@
+
+  <div ng-controller="ChatController">
+    <div class="chat-container is-down">
+      <div class="chat-header">
+        <div class="chat-logo">
+          <span ><sup ng-show="showNotifications">{{unreadMessages}}</sup></span>
+          <i class="material-icons">chat</i>
+           <span class="logo-t">Chat </span> 
+        </div>
+        <div class="up-down">
+          <i class="material-icons">{{open}}</i>
+        </div>
+      </div>
+      <div class="chat-body">
+        <div class="chat-messages">
+          <ul>
+            <li class="chat-li" ng-repeat="message in messages">
+              <!-- edit a message -->
+              <div class="user-img">
+                  <img src={{message.avatar}} alt="">
+              </div>
+              
+              <div class="chat-m">
+                <div class="chat-m-u">
+                  <h5>{{message.username}}</h5>
+                  <span>{{message.date}} </span>
+                </div>
+                <input ng-disabled="disable" class="text-message" ng-model="message.text" ng-change="messages.$save(message)"/>
+              </div>
+              <div ng-if="name == message.username" class="chat-s">
+                <!-- <i ng-click="edit()" class="material-icons">edit</i> -->
```

Comment:
If not needed (line 31) you may want to remove. Also, some of your class names are not descriptive enough(Line 23 and 24)

Reply:
Please attend to Moses's comment.

Reply:
thanks for the feedback, have made them more descriptive.